pyaedt/modules/Material.py

```diff
@@ -675,6 +685,38 @@ def __init__(self, parent, name, props=None):
                                                             MatProperties.get_defaultvalue(aedtname=property), None)
         pass
 
+    @property
+    def material_appearance(self):
+        """Material Appearance specified as an RGB list.
+
+        Returns
+        -------
+        type
+            Appearance of the material.
+        """
```

Comment:
Please include a basic example here along with the correct return type (looks to be a list).

Reply:
Done.